config/deployment/dev/dispatch.yaml

```diff
@@ -7,6 +7,8 @@ dispatch:
     service: brain
   - url: "osteo.dev.datadonationplatform.org/*"
     service: osteo
+  - url: "rare-x.dev.datadonationplatform.org/*"
+    service: rarex
```

Comment:
We have moved `dispatch.yaml` out to GCP's Secret Manager as a better shared space for this. I believe you should have access to the `dev` one (try [here](https://console.cloud.google.com/security/secret-manager/secret/app-engine-dispatch/versions?project=broad-ddp-dev)). If not, just ping me or Marco and we can get this in when it's time to deploy.

Reply:
Alright, I have access to these secrets. Can I ask you to update those secret's for Rare-X please? I'm not sure whether this is the only place where I have to change it.

Besides, A-T study is already in `develop` but I don't see any references to this study in secrets. Can you take a look please?

Thank you! :)

Reply:
Yup, I can do this. After updating the dispatch file in Secret Manager, we also need to update our DNS records if those are not updated yet (just need to do this once). I went ahead and took care of that. These URLs are live on the `dev` environment now, let me know if we need to change these:
```
https://atcp.dev.datadonationplatform.org
https://rare-x.dev.datadonationplatform.org
```

Reply:
Thank you!